gossip/keys.go

```diff
@@ -95,5 +95,5 @@ func MakeNodeIDKey(nodeID proto.NodeID) string {
 
 // MakeMaxAvailCapacityKey returns the gossip key for the given store's capacity.
 func MakeMaxAvailCapacityKey(nodeID proto.NodeID, storeID proto.StoreID) string {
-	return MakeKey(KeyNodeIDPrefix, nodeID.String(), storeID.String())
+	return MakeKey(KeyMaxAvailCapacityPrefix, nodeID.String(), storeID.String())
```

Comment:
@tschottdorf Is this what you intended here or did you mean for KeyNodeIDPrefix to be involved in all keys that start with node IDs? We didn't have any tests before where this mattered, but the new test added in this PR fails because MakeMaxAvailCapacityKey does not create keys that match KeyMaxAvailCapacityPrefix.